src/System.Diagnostics.Process/src/System/Diagnostics/PerformanceCounterLib.cs

```diff
@@ -118,7 +82,7 @@ internal byte[] GetPerformanceData(string item)
         {
             if (_performanceMonitor == null)
             {
-                lock (InternalSyncObject)
+                lock (s_internalSyncObject)
```

Comment:
Won't this fail with a null reference exception if s_internalSyncObject hasn't been initialized yet?  Or does something guarantee that it has been initialized?